colabs/analyzeFilmsInImdbDataset.ipynb

```diff
@@ -1,1142 +1,227 @@
 {
- "cells": [
```

Comment:
Są dwa rodzaje notatników. Takie którą służą do pracy i takie gdzie liczył się tylko wynik. 
To jest ten pierwszy rodzaj. W nim kasujemy outputy przed commitem.

Reply:
Output usunięty